waku/v2/node/waku_node.nim

```diff
@@ -44,7 +44,7 @@ import
 
 when defined(rln):
   import
-    ../protocol/waku_rln_relay/waku_rln_relay_types
+    ../protocol/waku_rln_relay/types
```

Comment:
Here we should also import the barrel import module

Reply:
Addressed in https://github.com/status-im/nwaku/commit/e9d15c538c906cba1b2a87d5fa31d301e991e2df

Reply:
You want this fix in this PR?

Reply:
I should have been more explicit regarding that 🙏🏼 

No, out of the scope of this PR. Here you have a draft PR for that. Would you mind taking it over? 😄